lib/routes.js

```diff
@@ -165,6 +165,34 @@ module.exports = ({ cache, config }) => {
     res.end()
   }
 
+  // Get files from latest release
+  exports.files = async (req, res) => {
+    for (const param of req.params) {
+      if (param.toLowerCase().includes('source')) {
```

Comment:
is there a way to check if this is even uploaded before we block it,

לא קריטי